HippUnfoldHCP/HippUnfoldHCP.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -eu
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+  # pipedirguessed=1
+   #fix this if the script is more than one level below HCPPIPEDIR
+   export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+
+
+opts_SetScriptDescription "Make some BIDS structures and run HippUnfold"
+
+opts_AddMandatory '--study-folder' 'StudyFolder' 'path' "folder containing all subjects"
+opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
```

Comment:
This was changed to Session during the refactoring?

Reply:
Same as the other comment.